src/Tmds.DBus.Tool/CodeGenCommand.cs

```diff
@@ -234,7 +234,7 @@ class CodeGenArguments
             public string OutputFileName { get; set; }
             public IEnumerable<string> SkipInterfaces { get; set; }
             public Dictionary<string, string> Interfaces { get; set; }
-            public List<string> Files { get; set; }
+            public IReadOnlyList<string> Files { get; set; }
```

Comment:
The List doesn't work with the `McMaster.Extensions.CommandLineUtils`'s package?
How did you find out?

Reply:
Compiler reports:
`Cannot implicitly convert type 'System.Collections.Generic.IReadOnlyList<string?>' to 'System.Collections.Generic.List<string>'. An explicit conversion exists (are you missing a cast?)`